part2/stages/Functions/install-main

```diff
@@ -531,6 +536,95 @@ install_bootloader_from_dom0fs()
     return 0
 }
 
+create_efi_boot_entries()
+{
+    local DISK_DEV="/dev/${TARGET_DISK}"
+
+    do_cmd efibootmgr -w -L "OpenXT (safe graphics)" -l "\EFI\BOOT\BOOTX64.EFI" -c -d ${DISK_DEV} --part 1 -u "openxt-support-safe-graphics" >&2
+    if [ "$?" != "0" ]; then
+        echo "Couldn't create EFI boot entry" >&2
+        return 1
+    fi
+
+    do_cmd efibootmgr -w -L "OpenXT (AMT)" -l "\EFI\BOOT\BOOTX64.EFI" -c -d ${DISK_DEV} --part 1 -u "openxt-support-amt" >&2
```

Comment:
You could cut down on the copy and pasting by looping over different options.

Reply:
True, although I think it is a bit more readable this way. Also, it is entirely possible that certain options will also require different flags to be passed to efibootmgr in the future, which would further complicate that loop.